Update instructions verifying chainctl.

**Describe the bug**
Just like #420 any usage of cosign should be updated.

One example is here:
https://edu.chainguard.dev/chainguard/chainguard-enforce/how-to-install-chainctl/#verifying-the-chainctl-binary-with-cosign

**To Reproduce**
Steps to reproduce the behavior:
1. Go to https://edu.chainguard.dev/chainguard/chainguard-enforce/how-to-install-chainctl/#verifying-the-chainctl-binary-with-cosign

2. run the step:
```shell
vaikas@vaikas-MBP cosign % COSIGN_EXPERIMENTAL=1 cosign verify-blob \
   --signature "https://dl.enforce.dev/chainctl/$(chainctl version 2>&1 |awk '/GitVersion/ {print $2}')/chainctl_$(uname -s | tr '[:upper:]' '[:lower:]')_$(uname -m).sig" \
   --certificate "https://dl.enforce.dev/chainctl/$(chainctl version 2>&1 |awk '/GitVersion/ {print $2}')/chainctl_$(uname -s | tr '[:upper:]' '[:lower:]')_$(uname -m).cert.pem" \
/usr/local/bin/chainctl

Error: verifying blob [/usr/local/bin/chainctl]: --certificate-identity or --certificate-identity-regexp is required for verification in keyless mode
main.go:74: error during command execution: verifying blob [/usr/local/bin/chainctl]: --certificate-identity or --certificate-identity-regexp is required for verification in keyless mode
```

**Expected behavior**
That it works :)

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - cosign v2.0

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/content/chainguard/chainguard-enforce/how-to-install-chainctl.md
+++ b/content/chainguard/chainguard-enforce/how-to-install-chainctl.md
@@ -95,16 +95,17 @@ If you received output that you did not expect, check your bash profile to make 
 You can verify the integrity of your `chainctl` binary using Cosign. Ensure that you have the latest version of Cosign installed by following our [How to Install Cosign guide](/open-source/sigstore/cosign/how-to-install-cosign/). Verify your `chainctl` binary with the following command:
 
 ```sh
-COSIGN_EXPERIMENTAL=1 cosign verify-blob \
+cosign verify-blob \
    --signature "https://dl.enforce.dev/chainctl/$(chainctl version 2>&1 |awk '/GitVersion/ {print $2}')/chainctl_$(uname -s | tr '[:upper:]' '[:lower:]')_$(uname -m).sig" \
    --certificate "https://dl.enforce.dev/chainctl/$(chainctl version 2>&1 |awk '/GitVersion/ {print $2}')/chainctl_$(uname -s | tr '[:upper:]' '[:lower:]')_$(uname -m).cert.pem" \
+   --certificate-identity "https://github.com/chainguard-dev/mono/.github/workflows/.release-drop.yaml@refs/tags/v$(chainctl version 2>&1 |awk '/GitVersion/ {print $2}')" \
+   --certificate-oidc-issuer https://token.actions.githubusercontent.com \
    $(which chainctl)
 ```
 
-You should receive output like the following:
+You should receive the following output:
 
 ```
-tlog entry verified with uuid: <uuid here> index: <log index here>
 Verified OK
 ```
 

